csrankings-h.csv

```diff
@@ -350,6 +350,8 @@ Harjinder Rahanu,Middlesex University,https://www.mdx.ac.uk/about-us/our-people/
 Harkeerat Kaur,IIT Jammu,https://www.iitjammu.ac.in/computer_science_engineering/faculty-list/~harkeeratkaur,Mt1kwb0AAAAJ
 Harley D. Eades III,Augusta University,https://metatheorem.org,Nbb0YZEAAAAJ
 Harley Eades III,Augusta University,https://metatheorem.org,Nbb0YZEAAAAJ
+Harm Griffioen,TU Delft,https://www.tudelft.nl/en/eemcs/the-faculty/departments/intelligent-systems/cybersecurityeemcs/people/harm-griffioen,fbUtrQcAAAAJ
+Harm J. Griffioen,TU Delft,https://www.tudelft.nl/en/eemcs/the-faculty/departments/intelligent-systems/cybersecurityeemcs/people/harm-griffioen,fbUtrQcAAAAJ

```

Comment:
Please use actual home page for both of these: https://harmgriffioen.github.io/

Reply:
I updated the webpage